include/kokkos-utils/tests/fixtures/callbacks/Manager.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef KOKKOS_UTILS_TESTS_FIXTURES_CALLBACKS_MANAGER_HPP
+#define KOKKOS_UTILS_TESTS_FIXTURES_CALLBACKS_MANAGER_HPP
+
+#include "kokkos-utils/callbacks/Manager.hpp"
+
+namespace Kokkos::utils::tests::fixtures::callbacks
+{
+//! Fixture for initializing and finalizing @ref Kokkos::utils::callbacks::Manager.
+class Manager
+{
+public:
+    Manager()  { Kokkos::utils::callbacks::Manager::initialize(); }
+    ~Manager() { Kokkos::utils::callbacks::Manager::finalize(); }
+};
+
+} // namespace Kokkos::utils::tests::fixtures::callbacks
```

Comment:
I wonder if this is still a "fixture" now that it's no longer really related to tests. Does this class now look more like a "scoped" class?

If that's the case, it's probably true for the `ExecutionSpaceInstance` too. We could rename the `tests/fixtures` as 
`tests/scoped`. And we could modify a bit this class like:  

```suggestion
namespace Kokkos::utils::tests::scoped::callbacks
{
//! Fixture for initializing and finalizing @ref Kokkos::utils::callbacks::Manager.
struct KOKKOS_ATTRIBUTE_NODISCARD Manager
{
    Manager()  { Kokkos::utils::callbacks::Manager::initialize(); }
    ~Manager() { Kokkos::utils::callbacks::Manager::finalize(); }

    Manager& operator=(const Manager&) = delete;
    Manager(const Manager&)            = delete;
};

} // namespace Kokkos::utils::tests::scoped::callbacks
```

Reply:
Done.